pages/ourteam.js

```diff
@@ -35,116 +38,128 @@ class Ourteam extends Component {
         const { isAuthenticated, name } = this.state;
 
         return (
-            <div style={{ backgroundImage: `url(${Background})`, maxWidth: '100%', height: '100%' }}>
+              <div style={{ maxWidth: '100%', height: '100%' }}>
```

Comment:
supernit: tabbing looks a bit off here, not sure if it is just on my end cause of the way github displays things tho

Reply:
Good catch. I was a little unsure too about what was going on.

✅ Fixed